packages/amplify-cli/src/extensions/amplify-helpers/read-json-file.js

```diff
@@ -7,8 +10,37 @@ function stripBOM(content) {
   return content;
 }
 
+class JSONParseError extends Error {
```

Comment:
I would recommend looking at an managed library as opposed to adding a custom JSONParseError class.

Reply:
Could you give me more specific suggestion?
I grepped `packages/amplify-cli` and I saw there is only one custom error class definition.
Are you suggesting that throwing generic error is good enough like `throw new Error("message")`?

Reply:
[parse-json](https://www.npmjs.com/package/parse-json) would be a good implementation here.

Reply:
It looks the same idea.
https://github.com/sindresorhus/parse-json/blob/master/index.js
https://github.com/zkat/json-parse-better-errors/blob/latest/index.js
Anyway, I will.

Reply:
Done